source/glossary.rst

```diff
@@ -275,7 +275,11 @@ Glossary
      A deployment uses a single external key to decrypt any object throughout the deployment.
 
      See also: :term:`SSE-KMS`, :term:`SSE-C`, :term:`encryption at rest`, :term:`network encryption`.
-   
+
+   standalone deployment
+     A :term:`single-node single-drive` (SNSD) MinIO deployment.
+     This term previously referred to the deprecated :ref:`Gateway and Filesystem Mode <minio-gateway-migration>` deployment types.
```

Comment:
Now that I think about it, seems odd to add a term for `gateway` since it's gone. But `standalone` is still in use to mean SNSD. Searching for gateway should turn up this and hopefully clarify.

Reply:
You have better context than I do. I'm still fuzzy on how exactly they are related.